apple/internal/local_provisioning_profiles.bzl

```diff
@@ -0,0 +1,80 @@
+"""
+Bazel rules for using locally installed provisioning profiles for debug builds
+"""
+
+def _local_provisioning_profiles(repository_ctx):
+    repository_ctx.symlink(
+        "{}/Library/MobileDevice/Provisioning Profiles".format(repository_ctx.os.environ["HOME"]),
+        "profiles",
+    )
+
+    repository_ctx.file("BUILD", """
+filegroup(
+    name = "profiles",
+    srcs = glob(["profiles/*.mobileprovision"]),
+    visibility = ["//visibility:public"],
+)
+
+filegroup(
+    name = "empty",
+    srcs = [],
+    visibility = ["//visibility:public"],
+)
+
+alias(
+    name = "fallback_profiles",
+    actual = "{}",
+    visibility = ["//visibility:public"],
+)
+""".format(repository_ctx.attr.fallback_profiles or ":empty"))
+
+local_provisioning_profiles = repository_rule(
+    environ = ["HOME"],
+    local = True,
```

Comment:
Should this be local? Caveat it's gotta run every time.

Because you symlink provisioning profiles directory into the external dir, it seems like enough to just glob the profiles for invalidation?

Reply:
Yea I think you're right!